libr/util/r_asn1.c

```diff
@@ -410,14 +412,13 @@ RASN1Object *r_asn1_create_object (const ut8 *buffer, ut32 length) {
 		} else {
 			object->list.objects = (RASN1Object**) calloc (count, sizeof (RASN1Object*));
 			if (!object->list.objects) {
-				r_asn1_free_object (&object);
+				RASN1_OBJ_FREE0 (object);
```

Comment:
is not necessary to always nullify the object, and if you do it, you can do that directly as object = NULL in the next line.